tests/Integration/Entity/UserTest.php

```diff
@@ -60,4 +61,14 @@ public function testItCanRetrieveAndRemoveAssignments(): void
         self::assertEmpty($subject->getAssignments());
         self::assertEmpty($subject->getAvailableAssignments());
     }
+
+    public function testItThrowsExceptionIfUserHasNoAssignments(): void
+    {
+        $this->expectException(AssignmentNotFoundException::class);
+        $this->expectExceptionMessage("User 'testUser' does not have any assignments.");
+
+        (new User())
```

Comment:
You will have to change it after this [PR](https://github.com/oat-sa/simple-roster/pull/211).

Reply:
Sure 👍